me.stonegate.tsacdop.yml

```diff
@@ -0,0 +1,36 @@
+app-id: "me.stonegate.tsacdop"
```

Comment:
I'm not sure about this app-id, this doesn't seem to be a real domain name which means it doesn't pass this part of the requirements https://github.com/flathub/flathub/wiki/App-Requirements#Application-ID `com.github.tsacdop` would work or `com.github.tsacdop.tsacdop-desktop`

Reply:
I can confirm that stonegate.me is my own doamin, I use it for my own projects.